src/plugins/score-plugin-scenario/Scenario/Document/Tempo/TempoView.hpp

```diff
@@ -56,8 +57,51 @@ class TempoPresenter final
       QObject* parent)
       : CurveProcessPresenter{style, layer, view, context, parent}
   {
+    // only disable the curve when using the first 2 inlets
+    for (int i = 0; i < 2; i++)
```

Comment:
I think that you can do it for all the inlets (as right now there are three afaik ?)

Reply:
The third inlet is position, witch should probably not affect the curve.